### PR TITLE
Added README.md

### LINKED ISSUE #1625: Add README.md for Array_Rotation
The file should contain the following:

 - Explanation
 -  Algorithm
  - Pseudocode
  - Example
  -  Time Complexity

Also add images/GIFs to explain if possible.

### PR DESCRIPTION
**Pull request by SimranMakhija7**
Adding README.md for Array Rotation

Fixes #1625 
#### Files Added:
- README.md

### CODE REVIEW

**Comment by jainaman224** [comment]
Please complete PR template properly.

File: Array_Rotation/README.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
+#Array Rotation
```
Comment (jainaman224) [inline Array_Rotation/README.md]: `# Array Rotation` is the right way to use Markdown. Please have a look at preview.

Comment (jainaman224) [reply Array_Rotation/README.md]: https://github.com/jainaman224/Algo_Ds_Notes/pull/2127/files?short_path=18564f4#diff-18564f4fa8ec225f109a0a6ba59f9efa

**Comment by SakshiShreya** [comment]
@SimranMakhija7 are you working on this?

**Comment by somya-kapoor** [comment]
@SimranMakhija7 are you working on this?

**Comment by somya-kapoor** [comment]
@SimranMakhija7 Dut to no activity, I am closing this issue.

**Pull request closed without merging**